server/api/users.js

```diff
@@ -8,6 +8,13 @@ router.get('/', (req, res, next) => {
     .catch(next);
 });
 
+//add user
+router.post('/adduser/', (req, res, next) => {
+  User.create(req.body)
+    .then(() => res.sendStatus(204))
```

Comment:
Don't you want to send the newUser after it is created?

```suggestion
    .then(newUser => res.send(newUser)
```